app/controllers/v0/sessions_controller.rb

```diff
@@ -213,5 +214,14 @@ def context_key
     rescue
       'unknown'
     end
+
+    def benchmark_tags(success = true)
+      tags = []
+      tags << "status:#{success ? 'success' : 'failure'}"
+      tags << "context:#{context_key}"
+      tags << "loa:#{@current_user.loa[:current]}" if @current_user
+      tags << "multifactor:#{@current_user.multifactor}" if @current_user
```

Comment:
is a tag value of `nil` a problem, or will statsd handle that okay? I don't think `loa` or `multifactor` ought to be nil in any of these callback paths but I want to make sure if they are they don't blow up here in the statsd code.

Reply:
If multifactor is nil, the tag will be `multifactor:nil`. If loa is nil, that might be an issue. Is that ever the case?

Reply:
The loa dict should not be nil, loa[:current] also shouldn't but I could see it happening. But in that case the tag will just be loa:nil. I think we're okay here.